### PR TITLE
Problems with noting the kind node image version

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
In the actual use of kurator, it does not specify the relationship between the version of kind and the node image used, which may lead to user errors when using it. So we added a note about it.

### CODE REVIEW

File: docs/content/en/boilerplates/prerequisites.md [hunk 1]
```diff
@@ -13,7 +13,10 @@ Deploy a kubernetes cluster using kurator's scripts.
 $ hack/local-dev-setup.sh
 reparing kind config in path: /tmp/tmp.xxxxx
 ...
-Local clusters is running.
+Local clusters is running. Note that the kind version and node image version need to be matched. 
+Now that kurator's local_dev_setup.sh is using the 1.25.3 node image. 
+You can refer to king's release info(https://github.com/kubernetes-sigs/kind/releases).
```
Comment (Xieql) [inline docs/content/en/boilerplates/prerequisites.md:18]:
```suggestion
You can refer to kind's release info(https://github.com/kubernetes-sigs/kind/releases).
```

**Comment by zirain** [comment]
why need this? I don't think it needs to be a 1-on-1 match.

**Comment by LiZhenCheng9527** [comment]
It doesn't need to be a 1-on-1 match, but in my actual use of kurator, I got an error when creating a cluster using the 1.25.3 version of the node image. The reason is because the node image versions don't match. So I feel the need to clarify.

File: docs/content/en/boilerplates/prerequisites.md [hunk 2]
```diff
@@ -13,7 +13,10 @@ Deploy a kubernetes cluster using kurator's scripts.
 $ hack/local-dev-setup.sh
 reparing kind config in path: /tmp/tmp.xxxxx
 ...
-Local clusters is running.
+Local clusters is running. Note that the kind version and node image version need to be matched. 
+Now that kurator's local_dev_setup.sh is using the 1.25.3 node image. 
+You can refer to kind's release info(https://github.com/kubernetes-sigs/kind/releases).
```
Comment (hzxuzhonghu) [inline docs/content/en/boilerplates/prerequisites.md:18]: Can you be more specific, if there is a mismatch maybe we need to have a support matrix like this https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases

Comment (LiZhenCheng9527) [reply docs/content/en/boilerplates/prerequisites.md:18]: Kurator now uses the kindest/node image from version 1.25.3, which works fine with king versions 0.16.0-0.19.0. It does not work with version 0.20.0 and 0.15.0 and earlier.

**Pull request closed without merging**